data/json/monsters/zed_nomex.json

```
@@ -0,0 +1,78 @@
+[
+  {
+    "id": "mon_zombie_nomex",
+    "type": "MONSTER",
+    "name": { "str": "nomex zombie" },
+    "description": "this zombie was a firefighter in its past life, still donned in its turnout equipment - said equipment has melded with its flesh, with the tough, flame resistant fibers becoming incorporated into its skin as armor and body mass.",
+    "looks_like": "mon_zombie_fireman",
+    "copy-from": "mon_zombie_fireman",
+    "color": "yellow",
+    "armor": { "bash": 12, "cut": 16, "acid": 14, "heat": 18, "bullet": 10, "electric": 4 },
+    "harvest": "zombie_nomex",
+    "hp": 100,
+    "upgrades": { "half_life": 32, "into_group": "GROUP_FIREMAN_UPGRADE" }
+  },
+  {
+    "id": "mon_zombie_fireproof",
+    "type": "MONSTER",
+    "name": { "str": "fireproof zombie" },
+    "description": "this zombie appears to have once been a firefighter or similar first responder, the wilted remains of its turnout gear cling to it like a molted carapace.  Fibrous streaks resembling the fireproof materials of its gear run through its flesh like roots, giving the thing a mottled, off-yellow appearance as it prowls on all fours.",
+    "copy-from": "mon_zombie_hunter",
+    "default_faction": "zombie",
+    "color": "brown_yellow",
+    "symbol": "Z",
+    "hp": 120,
+    "armor": { "bash": 15, "cut": 18, "acid": 14, "heat": 22, "bullet": 11, "electric": 4 },
+    "harvest": "zombie_nomex",
+    "upgrades": false,
+    "death_drops": "mon_zombie_fireman_death_drops",
+    "extend": {
+      "weakpoint_sets": [ "wps_humanoid_turnout_gear", "wps_humanoid_full_helmet" ],
+      "families": [ "prof_wp_syn_armored", "prof_wp_nat_armored" ],
+      "flags": [ "ACIDPROOF", "FIREPROOF", "CLIMBS" ]
+    },
+    "delete": { "special_attacks": [ "bite_humanoid" ] }
+  },
+  {
+    "id": "mon_zombie_fireproof_grappler",
+    "type": "MONSTER",
+    "name": { "str": "fireproof grappler" },
+    "description": "this zombie appears to have once been a firefighter or similar first responder, its body having ripped through and absorbed the majority of its uniform.  Its torso and back are swollen, and its muscles are marbled with knotted growths resembling the nomex fibers of its former gear, feeding into lanky arms that drag on the floor.",
```

```suggestion
    "description": "This zombie appears to have once been a firefighter or similar first responder, its body having ripped through and absorbed the majority of its uniform.  Its torso and back are swollen, and its muscles are marbled with knotted growths resembling the Nomex fibers of its former gear, feeding into lanky arms that drag on the floor.",
```
*Angry DuPoint noises*